apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-async-example/multi-input-async-example.component.html

```diff
@@ -1,4 +1,7 @@
+<label fd-label for="multiInputAsync">Multi Input with async data</label>
 <fd-multi-input
+    inputId="multiInputAsync"
+    [title]="'search'"
```

Comment:
doing `title='search'` will show `tooltip` on input as well, which is not needed. this title input is to show `tooltip` on add-on button only. 
It is causing issue while screen reading. so have to bind here.

Reply:
I guess in that case we need to write about that fact in the typeDoc for the property.